File: vue3/src/api/message.js
import request from '@/utils/request.js'

const selectMessageDataAPI = (
  pageNum,
  pageSize,
  sendId,
  sendRole,
  receiverId,
  receiverRole,
  paperId,
) => {
  return request.get('/message/selectByPage', {
    params: { pageNum, pageSize, sendId, sendRole, receiverId, receiverRole, paperId },
  })
}

export { selectMessageDataAPI }

File: springboot3/papaer/src/main/java/com/paper/entity/Message.java
package com.paper.entity;

import com.paper.common.enums.Role;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

/**
 * @author 林万奇
 * @since 2025-02-27
 */
@Getter
@Setter
public class Message {
    private Integer id;
    private Integer sendId;
    private Role sendRole;
    private Integer receiverId;
    private Role receiverRole;
    private Integer paperId;
    private String content;
    private LocalDateTime createTime;
}
